tpl/js-templates.php

```diff
@@ -519,7 +519,7 @@
 		</form>
 
 		<div class="so-sidebar-tools">
-			<a href="#" class="live-editor-close" title="<?php esc_attr_e('Close Live Editor', 'siteorigin-panels') ?>"></a>
+			<button href="#" class="live-editor-close button-primary"><?php esc_attr_e('Done', 'siteorigin-panels') ?></button>
```

Comment:
Same as before. Necessary to have `esc_attr_e` and not just `_e`? Otherwise, I guess it should be `esc_html_e`?